utils/spdx/build.gradle.kts

```diff
@@ -66,8 +65,7 @@ val importScanCodeLicenseTexts by tasks.registering(SvnExport::class) {
     description = "Imports license texts from the ScanCode repository."
     group = "SPDX"
 
-    val versionWithoutHyphen = scancodeVersion.replace("-", "")
-    svnUrl = "https://github.com/nexB/scancode-toolkit/tags/v$versionWithoutHyphen/src/licensedcode/data/licenses"
+    svnUrl = "https://github.com/nexB/scancode-toolkit/trunk/src/licensedcode/data/licenses"
```

Comment:
> Since the ScanCode license texts are now only used for the SPDX licenses

Huh? So ScanCode license texts are not also used for ScanCode licenses (licenses only found by ScanCode)?!

Reply:
Yes, as we do not import LicenseRef-scancode license texts anymore the texts imported from ScanCode are only used for SPDX licenses since the previous commits.